src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -1,7 +1,7 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import {StyleSheet, Text, TouchableNativeFeedback, TouchableOpacity, View} from 'react-native';
-import {fonts} from '../../../utils/fonts';
+import {fonts, normalize} from '../../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to include the `normalize` function from the `fonts` module. Ensure that this function is used appropriately in the code and doesn't introduce any side effects.
```diff
- import {fonts} from '../../../utils/fonts';
+ import {fonts, normalize} from '../../../utils/fonts';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->